src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -754,8 +754,13 @@ do so, use `canMakePayment()`.
       request.show();
     }
 
-Note: This is only optional. Users can still add a new payment method in the
-Payment Request UI in general.
+Warning: In order to protect user privacy, invocation of <code>canMakePayment()
+</code> API is rate limited in Google Chrome. You'll receive "Quota exceeded"
```

Comment:
If the rate limiting is not spec'd can we call it out as differing between each browser?

Can we include any indication of what the rate limit is? I.e. X number of times in Y amount of time